<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quadritic_gui</class>
 <widget class="QDialog" name="quadritic_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EquateToGraph - Quadritic Function</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../equatetograph.qrc">
    <normaloff>:/images/images/title_icon.png</normaloff>:/images/images/title_icon.png</iconset>
  </property>
  <widget class="QLabel" name="lbl_enter_values">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter the Constant Values</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_quadOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_quadep_form">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quadritic Equation Form</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_quadeq">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y = ax^2 + bx + c</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_funcDetail">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Function Details</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_axisOfSym">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Axis of Symmetry</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_discValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Discriminant Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_graphDraw">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Draw Graph</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_aValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_aValue"/>
    </item>
    <item>
     <widget class="QLabel" name="lbl_bValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>b</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_bvalue"/>
    </item>
    <item>
     <widget class="QLabel" name="lbl_CValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>c</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineCValue"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clear">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>361</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_saveImage">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save As an Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../equatetograph.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>text_quadOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_aValue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_bvalue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>lineCValue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
